types/numeral/test/numeral-tests-global.ts

```diff
@@ -0,0 +1,102 @@
+const valueFormat: string = numeral(1000).format('0,0');
```

Comment:
Oh, usually it is just literally same file as copy CJS test, minus removed header imports and it should work same way.
And yes, it could be as short, as veryfing that this global reference is of expected type (e.g. `// $ExpectType SomeKindOfStaticType`
So, do you want me to amend this part?

Reply:
I don't mind duplication, some other packages did this as well